pull_request_template.md

```diff
@@ -13,10 +13,14 @@
 - [ ] 3.3.5
 - [ ] master
 
-**Issues addressed:** Closes #  (insert issue tracker number)
+**Issues addressed:**
 
```

Comment:
why extra empty line?

Reply:
Because GitHub does not seem to read the Close command properly when it follows other text on the same line.
My previous PR did not get recognized as having the function to close a mentioned issue, because the Close command was not on its own line.

Reply:
Oh. You mean the blank line between the bold text and the following line.
Well, that is more for readability than for anything else. It just looks better.

Reply:
@illfated I _think_ you are confusing the `Closes #....` in a commit with this one here. This one does not trigger any action on GitHub's side.
Only "commands" in commits are considered. For the master branch GitHub does it on it's own, for 335 it's the GitHub action jackpoz wrote (https://github.com/TrinityCore/GitHub-Actions/commits/issue-closer).

@jackpoz I'm not telling stupid things, right?

Reply:
Fair enough. I guess I should build my own commit generator instead of copy-pasting the one on GitHub.

Reply:
@illfated Thanks for pinging me on the other issue. I stand corrected :)

I'm open to getting this merged.